_posts/2015-12-04-keep-yourself-sane-with-motd-ascii-art.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+title:  "Keep Yourself Sane, Tweek your MOTD"
+date:   2015-12-04 21:10:00
+categories: linux motd
+author: admbradford
+---
+
+While doing server administration 3 AM I've made my fair share of mistakes. Most were a result of being on the wrong server. Several years ago, fed up and full of 4 AM fury, I placed Angry Axe Homer into a MOTD file.
+
+A few days later my passion for logging into the wrong server was greeted by:
+
+<pre style="font-family:'Lucida Console', 'Courier New', 'Courier', monospace; font-size:9px; background-color:#000; color:#fff; line-height:1;">
```

Comment:
I'm wondering if using something like this wouldn't work instead of using inline-styles. Mind checking it out?

\`\`\`
beautiful
art goes
here
\`\`\`

That should transform into something like this:
```
beautiful
art goes
here
```